api/api/models/mission.py

```diff
@@ -0,0 +1,24 @@
+"""This module manages generic models required for missions
+
+The following models are present here:
+
+    - Auth: Mission
+"""
+from django.db import models
+
+from api.models import Auth, MAX_TITLE_LENGTH
+
+
+# TODO: add a Team one model will be available
+# Create role Manager ??
+class Mission(models.Model):
+    # Duration is in number of days.
+    duration = models.FloatField(default=1)
```

Comment:
why is this a `FloatField` ? I'm not sure to understand the purpose of this property

Reply:
ahhhh, so it's the number of days / half-days ? If that's the case, maybe we should rename this field lol